subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/util/View.kt

```diff
@@ -4,13 +4,18 @@ import android.graphics.Rect
 import android.view.View
 import android.view.ViewGroup
 import android.view.ViewParent
-import androidx.test.espresso.util.TreeIterables
 
-internal fun View.getAboveVisibleViews(): List<View> {
-    return TreeIterables.breadthFirstViewTraversal(this.rootView)
-        .dropWhile { it != this }
-        .drop(1)
-        .filter { it.visibility == View.VISIBLE }
+internal fun View.forEachVisibleViewsWithHigherZOrder(block: (View) -> Boolean): Boolean {
```

Comment:
Could you add comment with link to TreeIterables.breadthFirstViewTraversal(this.rootView)
And describe that it's a better version because of something

Reply:
Done